composer/modules/web/src/plugins/import-struct/handlers.js

```diff
@@ -115,15 +115,21 @@ export function getHandlerDefinitions(plugin) {
 
                 const onImport = (json, structName, removeDefaults) => {
                     let success = true;
+
+                    if (structName !== undefined && structName !== ''){
```

Comment:
What if we use _if (structName !== undefined && structName !== ''){_ instead of using !==? Just a suggestion
:)

Reply:
Good suggestion. Updated code with d4c0d9765f1a4e6f9c9a2efd89e381ea9e54db2a